drivers/scheduler/k8s/specs/test-sv4-svc-2/pod.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: test-sv4-2-dep-svc
```

Comment:
is it possible to have a different name other than test-sv4-2-. ?

Reply:
yeah, I don't like that name as well. But I could not come up with something better. This app is similar to test-sv4-svc but with slightly different settings. We may want to add more different settings in the future. Do you have any suggestion?

Reply:
yeah i don't know if there's a better name for it now. In the future, i think it's possible to parameterize the different settings. But it might take some time to do it.